src/test/goldens/polymer/lib/utils/mixin.d.ts

```diff
@@ -18,5 +18,5 @@ declare namespace Polymer {
    * if it has not already been applied its base argument. Also memoizes mixin
    * applications.
    */
-  function dedupingMixin(mixin: T|null): any;
+  function dedupingMixin(mixin: T|null): T|null;
```

Comment:
?

Reply:
It's because I updated the annotation on my Polymer PR, which the golden generation pulls from. After rebase onto https://github.com/Polymer/gen-typescript-declarations/pull/34 this will go away.